src/api/cvc4cpp.cpp

```diff
@@ -5678,6 +5678,37 @@ SmtEngine* Solver::getSmtEngine(void) const { return d_smtEngine.get(); }
  */
 Options& Solver::getOptions(void) { return d_smtEngine->getOptions(); }
 
+/* -------------------------------------------------------------------------- */
+/* Optimization                                                               */
+/* -------------------------------------------------------------------------- */
+
+Objective::Objective(Term t, ObjectiveType type):
```

Comment:
Remove one colon

Reply:
or more generally, make sure it compiles and passes clang-format.